src/Uno.UI/UI/Xaml/Thickness.cs

```diff
@@ -41,6 +41,8 @@ public Thickness (double leftRight, double topBottom)
 
 		public double Bottom { get ; set; }
 
+		internal Thickness GetInverse() => new Thickness(-Left, -Top, -Right, -Bottom);
```

Comment:
Is there a reason to not do this as an extension method ?

Reply:
It's internal... so no reason except to avoid spreading the code uselessly.